MH4U Dex/MonsterDetailsViewController.m

```diff
@@ -29,39 +34,61 @@ @implementation MonsterDetailsViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
-    [self.lowRankSubView setHidden:YES];
+    self.overviewSubView.hidden = NO;
+    self.lowRankSubView.hidden = YES;
+    self.highRankSubView.hidden = YES;
+    self.GRankSubView.hidden = YES;
 }
 
 - (IBAction)segmentedControlChanged:(UISegmentedControl *)segmentedControl
 {
     switch (segmentedControl.selectedSegmentIndex) {
         case Overview:
-            [self.overviewSubView setHidden:NO];
-            [self.lowRankSubView setHidden:YES];
+            self.overviewSubView.hidden = NO;
+            self.lowRankSubView.hidden = YES;
+            self.highRankSubView.hidden = YES;
+            self.GRankSubView.hidden = YES;
             self.navigationItem.title = [NSString stringWithFormat:@"%@ Overview", self.monster];
             break;
         case LowRank:
-            [self.overviewSubView setHidden:YES];
-            [self.lowRankSubView setHidden:NO];
+            self.overviewSubView.hidden = YES;
+            self.lowRankSubView.hidden = NO;
+            self.highRankSubView.hidden = YES;
+            self.GRankSubView.hidden = YES;
             self.navigationItem.title = [NSString stringWithFormat:@"%@ LR Drops", self.monster];
```

Comment:
Check into the difference between setting `self.navigationItem.title` and `self.title` to make sure that you're setting the correct one.